### DISCUSSION TITLE
Max Window Width

### DISCUSSION THREAD

**Discussion by mike-lloyd03** [hyprland-community/pyprland#139] [General]
I'd like to set a maximum window width so tiled windows don't take up the whole screen on my ultrawide.

This has been discussed before on Hyprland (https://github.com/hyprwm/Hyprland/issues/2092). A solution was recommended there which uses gaps to mimic this behavior.

This works well while my laptop is connected to my ultrawide, but if I undock, I have to update my config to remove the gaps.

Is it possible for me to write a pyprland plugin that would do something similar?

**Comment by fdev31** [comment]
Please check https://hyprland-community.github.io/pyprland/scratchpads_advanced.html#max-size - it's not helping ? I shared my dotfiles which are making use of it (I had the same problem).
